jabgui/src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogPreferences.java

```diff
@@ -7,12 +7,21 @@
 
 import org.jabref.logic.externalfiles.DateRange;
 import org.jabref.logic.externalfiles.ExternalFileSorter;
+import org.jabref.logic.util.StandardFileType;
 
 public class UnlinkedFilesDialogPreferences {
     private final StringProperty unlinkedFilesSelectedExtension;
     private final ObjectProperty<DateRange> unlinkedFilesSelectedDateRange;
     private final ObjectProperty<ExternalFileSorter> unlinkedFilesSelectedSort;
 
+    private UnlinkedFilesDialogPreferences() {
+        this(
+                StandardFileType.ANY_FILE.getName(),
+                DateRange.ALL_TIME,
+                ExternalFileSorter.DEFAULT
+        );
+    }
```

Comment:
Missing comments here. Please create another PR and add comments to these values.